src/core/core.controller.js

```diff
@@ -35,8 +35,13 @@ defaults._set('global', {
 	responsiveAnimationDuration: 0
 });
 
-function mergeScaleConfig(/* objects ... */) {
-	return helpers.merge(helpers.clone(arguments[0]), [].slice.call(arguments, 1), {
+/**
+ * Recursively merge the given config objects as the `scales` options by
```

Comment:
It might be clearer to say "`scales`  config objects" rather than "config objects as the `scales` options"

Reply:
The idea is that this method considers the given config objects representing the `scales` property located at the root of the chart options, so not sure your way is clearer, though I agree that mine is also ambiguous. What about the following:

> Recursively merge the given config objects as the `scales` property of the chart options by...

Reply:
Either your original or latest suggestion sound fine to me if we swap "as" for "representing"